Endava.TechCourse.BankApp.Application/Commands/DeleteWalletType/DeleteWalletTypeCommand.cs

```diff
@@ -0,0 +1,8 @@
+using MediatR;
+
+namespace Endava.TechCourse.BankApp.Application.Commands.DeleteCurrency;
+
+public class DeleteWalletTypeCommand : IRequest<CommandStatus>
+{
+    public Guid Id { get; set; }
```

Comment:
Better to have string in your commands for Id's and to tryParse it inside the command handler